queries.sql

```diff
@@ -7,4 +7,94 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, date_of_birth FROM animals WHERE weight_kg >= '10.5';
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Inside a transaction update the animals table by setting the species column to unspecified */
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+COMMIT;
+
+/* Update the animals table by setting the species column to digimon for all animals that have a name ending in mon */
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+/* To verify changes */
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+COMMIT;
+
+/* To delete all records in a transaction */
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+COMMIT;
```

Comment:
For this transaction, it is not necessary to `COMMIT` the changes.

Reply:
done